### PR TITLE
i guess ive created what youve asked 

### PR DESCRIPTION
**Pull request by belikegt**
please let me know if this is what you asked for

### CODE REVIEW

File: Giftson/commandLineFunctionPassing.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+""" Simple Python program to use command line argument to call a function with parameters """
```
Comment (Rangeeshar) [inline Giftson/commandLineFunctionPassing.py]: Add how to run this file command here.

**Review by Rangeeshar — CHANGES_REQUESTED** [review]
changes required

File: Giftson/commandLineFunctionPassing.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+""" '
```
Comment (Rangeeshar) [inline Giftson/commandLineFunctionPassing.py]: what is this ' ??? doing here

**Review by belikegt — COMMENTED** [review]
changes made

**Review by Rangeeshar — CHANGES_REQUESTED** [review]
changes

File: Giftson/commandLineFunctionPassing.py (hunk 2 above)
Comment (belikegt) [reply Giftson/commandLineFunctionPassing.py]: sorry typo 😅😅

**Review by belikegt — COMMENTED** [review]
updated

**Review by belikegt — COMMENTED** [review]
done

**Review by Rangeeshar — APPROVED** [review]

**Pull request merged**